content/docs/reference/config.mdx

```diff
@@ -56,3 +56,37 @@ export default defineConfig({
 
 In this example, the TinaCMS admin will be viewable at `<my-site-url>/admin/index.html`.
 For more information [check out the content modeling section](/docs/schema/).
+
+
+## Typescript Path Alias 
+
+TinaCMS supports TypeScript path aliases in the `tina.config.js` file and collections, making it easier to organize and maintain your code.
+
+##### Use alias in config.ts
+
+Ensure that `tsconfig.json` exists at the root of your project. Define your aliases in the `compilerOptions` section:
+
+```json
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+      "paths": {
+      "@/*": ["src/*"]
+    }
+  }
+}
+```
+
+Figure: Your defined `tsconfig.json`
+
+You can now import utilities or other modules using the aliases:
+
+```typescript
+import { someUtility } from '@/lib/utils';
+
+export default defineConfig({
+  ....
+})
+```
+
+Figure: Using aliases in tina `config.ts`
```

Comment:
**issue**: The "tina" word should either be part of the path or be capitalized.

```suggestion
Figure: Using aliases in `tina/config.ts`
```

Reply:
Resolved with rewording to tina/config.{ts,js,tsx}